### PR TITLE
Add New Relic, configure artillery for stress testing

### PR DESCRIPTION
**Pull request by ryancmoniz**

### CODE REVIEW

File: artillerygen.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  const description = Faker.lorem.paragraph();
+
+  userContext.vars.id = id;
+  userContext.vars.hours = hours;
+  userContext.vars.rating = rating;
+  userContext.vars.description = description;
+  userContext.vars.user_id = user_id;
+
+  return done();
+}
```
Comment (ingridclaire) [inline artillerygen.js:24]: what will you do with this generated review? Are these the id's and other info that you will be querying to stress test?

File: newrelic.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      'request.headers.cookie',
+      'request.headers.authorization',
+      'request.headers.proxyAuthorization',
+      'request.headers.setCookie*',
+      'request.headers.x*',
+      'response.headers.cookie',
+      'response.headers.authorization',
+      'response.headers.proxyAuthorization',
+      'response.headers.setCookie*',
+      'response.headers.x*'
```
Comment (ingridclaire) [inline newrelic.js:51]: is this stuff in the new relic documentation? was it straightforward figuring out how to configure everything?

File: artillerygen.js (hunk 1 above)
Comment (ryancmoniz) [reply artillerygen.js:24]: This is random data that I want artillery to use if it makes post requests. I didn’t want to just spam one identical record 1000 times into my database.

File: newrelic.js (hunk 2 above)
Comment (ryancmoniz) [reply newrelic.js:51]: Yah with new relic you copy the newrelic.js file from node modules into your main repo folder and just change the name and the key inside it. Really straightforward.

**Pull request merged**